packages/color-kit-block/src/ColorKitBlock.tsx

```diff
@@ -0,0 +1,52 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import 'tailwindcss/tailwind.css';
+import '@frontify/fondue-tokens/styles';
+
+import { FC, ReactElement } from 'react';
+import { useBlockSettings, useColorPalettes } from '@frontify/app-bridge';
+import { Badge, Button, ButtonStyle, IconArrowCircleDown, Text } from '@frontify/fondue';
+
+import { Palette } from './Palette';
+import type { ColorKitBlockProps, Settings } from './types';
+
+export const ColorKitBlock: FC<ColorKitBlockProps> = ({ appBridge }): ReactElement => {
+    const [blockSettings] = useBlockSettings<Settings>(appBridge);
+
+    const { colorPalettes, getDownloadColorKitLink } = useColorPalettes(appBridge, blockSettings.colorPalettes);
+    const link = getDownloadColorKitLink(blockSettings.colorPalettes);
```

Comment:
`useMemo`

Reply:
I think there is no need for memoization as getDownlaodColorKitLink is not an expensive function and returns a simple string (no change in reference).